website/draft_site/drafts/models.py

```diff
@@ -1,3 +1,38 @@
 from django.db import models
 
-# Create your models here.
+
+class Draft(models.Model):
+    num_drafters = models.IntegerField()
+    num_phases = models.IntegerField()
+    cards_per_pack = models.IntegerField()
+
+    def __str__(self):
+        return str(self.__dict__)
+
+
+class Drafter(models.Model):
+    draft = models.ForeignKey(Draft, on_delete=models.CASCADE)
+    seat = models.IntegerField()
+    # 0-indexed
+    current_phase = models.IntegerField(default=0)
+    # 0-indexed
+    current_pick = models.IntegerField(default=0)
```

Comment:
To me it seems like current_phase and current_pick belong to the draft and not the drafter?